apps/librelingo_yaml_loader/tests/testing_utilities.py

```diff
@@ -0,0 +1,19 @@
+"""Contains utilities for tests of the librelingo_yaml_loader"""
+
+import random
+
+
+def get_some_int():
```

Comment:
```suggestion
def get_random_int():
```

Reply:
btw there's already a package called librelingo fakes. And I think it already has these generators :thinking:

Reply:
at least, there's something similar: https://github.com/kantord/LibreLingo/blob/main/apps/librelingo_fakes/librelingo_fakes/fakes.py#L270

Reply:
In few places the `random_path` used to be a [`string`](https://github.com/kantord/LibreLingo/blob/37a04d004cbc341f0ffe8b256c1f5a6a168539ad/apps/librelingo_yaml_loader/tests/test_yaml_loader.py#L986) and now it is a proper [`pathlib.Path`](https://github.com/vil02/LibreLingo/blob/1bfccff9fc72d5ea1e1da3a0d2f3199283fea038/apps/librelingo_yaml_loader/tests/test_yaml_loader.py#L828). In my opinion it was meant to be like this, but please have a look.

Reply:
I am not sure what you mean, what does the path thing have to do with string? :thinking:  sorry I'm a bit confused.

in any case I don't mean that you have to use that exact one. You can just add a new fake in that library, I think it would be a good idea to be able to share it between all packages.

Reply:
In the old code, the variable `randomPath`
https://github.com/kantord/LibreLingo/blob/37a04d004cbc341f0ffe8b256c1f5a6a168539ad/apps/librelingo_yaml_loader/tests/test_yaml_loader.py#L986
was a `string` like `'123'`.
Now `random_path`
https://github.com/vil02/LibreLingo/blob/1bfccff9fc72d5ea1e1da3a0d2f3199283fea038/apps/librelingo_yaml_loader/tests/test_yaml_loader.py#L828
is a `pathlib.Path` like `pathlib.Path('./123')`.
I think the current situation makes more sense: the variable was `randomPath` not `randomFolderName`. And the behaviour is the same: tested functions converts the input into `pathlib.Path` anyway.

Reply:
ok, that makes sense